src/olympia/users/tests/test_forms.py

```diff
@@ -214,6 +214,98 @@ def test_cannot_change_email(self):
         form.save()
         assert self.user.reload().email == 'me@example.com'
 
+    def test_only_show_notifications_user_has_permission_to(self):
+        with patch('basket.base.request', autospec=True) as request_call:
+            request_call.return_value = {
+                'status': 'ok', 'token': '123', 'newsletters': []}
+
+            # 8 is the `announcements` notification, or about-addons newsletter
```

Comment:
You're passing an empty dict here, so the comment is just confusing

Reply:
Nice catch, that's an old comment, removed in be02258db5